cl/opinion_page/templates/includes/de_list.html

```diff
@@ -148,24 +165,17 @@
                             </span>
                     {% else %}
                       {% if rd.pacer_url %}
-
-                        {% if not request.COOKIES.buy_on_pacer_modal and not request.COOKIES.recap_install_plea %}
-                          <a href="{{ rd.pacer_url }}"
-                          data-toggle="modal" data-target="#modal-buy-pacer"
-                          target="_blank" class="open_buy_pacer_modal btn btn-default btn-xs"
+                        <a href="{{ rd.pacer_url }}"
+                          {% if not request.COOKIES.buy_on_pacer_modal and not request.COOKIES.recap_install_plea %}
+                            data-toggle="modal" data-target="#modal-buy-pacer"
+                            class="open_buy_pacer_modal btn btn-default btn-xs"
+                          {% else%}
+                            class="btn btn-default btn-xs"
+                          {% endif %}
+                          target="_blank"
                           rel="nofollow"
-                          title="Buy on PACER {% if rd.page_count %}(${{ rd|price }}){% endif %}"><i class="fa fa-download"></i></a>
-
-                        {% else%}
-
-                          <a href="{{ rd.pacer_url }}"
-                           class="btn btn-default btn-xs"
-                           target="_blank"
-                           rel="nofollow"
-                           title="Buy on PACER {% if rd.page_count %}(${{ rd|price }}){% endif %}"><i class="fa fa-download"></i></a>
-
-                        {% endif %}
-
+                          title="Buy on PACER {% if rd.page_count %}(${{ rd|price }}){% endif %}"><i class="fa fa-download"></i>
```

Comment:
This if statement isn't needed right?

Reply:
or I see we need it for the ($  )?

Reply:
yeah, it's used to only show the price inside ($ ) if we have the `page_count`